site/gdocs/components/Byline.tsx

```diff
@@ -0,0 +1,39 @@
+import { getCanonicalUrl } from "@ourworldindata/components"
+import { OwidGdocType } from "@ourworldindata/types"
+import React from "react"
+import { useLinkedAuthor } from "../utils.js"
+
+export const Byline = ({ names }: { names: string[] }) => {
+    return (
+        <>
+            {"By: "}
+            {names.map((name, idx) => (
+                <React.Fragment key={name}>
+                    <LinkedAuthor name={name} />
+                    {idx === names.length - 1
+                        ? ""
+                        : idx === names.length - 2
+                          ? " and "
+                          : ", "}
+                </React.Fragment>
+            ))}
+        </>
+    )
+}
+
+const LinkedAuthor = ({ name }: { name: string }) => {
+    const author = useLinkedAuthor(name)
+    // Somehow, using a fragment here messes up the hydration process and causes
+    // some links to be applied to the wrong authors, but only if the first
+    // author is unlinked (which probably gets tangled with the "By: " text
+    // above). Additional markup, here in the form of a span, works more
+    // reliably.
+    if (!author.slug) return <span>{author.title}</span>
```

Comment:
Just wondering if here we still want to link to the team page as a fallback? Not sure if a decision has already been made here but are we even going to be keeping those pages around? 🤔

Reply:
yes we might want to do that in case we're using the fallback "Our World in Data team". But for external contributors (which is rare, granted), this wouldn't be desirable in my opinion. I'd go for wait and see on this one.